packages/resource-deployment/scripts/function-app-create.sh

```diff
@@ -151,6 +151,18 @@ function deployWebWorkersFunction {
     deployFunctionApp "web-workers-allyfuncapp" "${0%/*}/../templates/function-web-workers-app-template.json" "$webWorkersFuncAppName"
 }
 
+function restrictWebApiAccess {
```

Comment:
Just for learning purposes: is there a particular reason y'all opted to edit programmatically rather than leverage the widget in Azure portal? Or does the Azure portal solution not support everything we need to fix the issue?

Reply:
We deploy new resource groups a lot for testing purposes, and it's easier if everything we need gets set up by the deployment scripts without having to go to the portal and change settings manually (plus it helps keep everything documented)